assets/acquia_lift.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @file
+ * Contains the definition of the behaviour openAcquiaLiftExperienceBuilder.
+ */
+
+(function ($, Drupal) {
+
+    'use strict';
+
+    /**
+     * Attaches the JS test behavior to to weight div.
+     */
+    Drupal.behaviors.acquiaLiftOpenExperienceBuilder = {
+        attach: function (context, settings) {
+            $('.openLiftLink').click(function() {
```

Comment:
Also, it's a best practice to have the actual logic in separate functions, for maintainability reasons.

So, all feedback combined, I'd propose something like

```
/**
 * Opens the Acquia Lift Experience Builder, if it is available.
 */
function openExperienceBuilder() {
  if (window.AcquiaLiftPublicApi) {
    window.AcquiaLiftPublicApi.activate();
  }
  else {
    alert('Sorry, Acquia Lift could not be found');
  }
}

Drupal.behaviors.acquiaLiftOpenExperienceBuilder = {
  attach: function (context, settings) {
    var trigger = document.querySelector('.toolbar .toolbar-bar .openLiftLink');
    trigger.addEventListener('click', openExperienceBuilder);
  }
};
```